### PR TITLE
python3: Update to 3.10.2, refresh patches

### PR DESCRIPTION
**Pull request by jefferyto**
Maintainer: me
Compile tested: armvirt-32, 2022-01-10 snapshot sdk
Run tested: none

Description:
Signed-off-by: Jeffery To <jeffery.to@gmail.com>

### CODE REVIEW

File: lang/python/python3/Makefile [hunk 1]
```diff
@@ -206,14 +205,6 @@ define Build/Compile/python3-pip
 		--root=$(PKG_BUILD_DIR)/install-pip \
 		--prefix=/usr \
 		$(PKG_BUILD_DIR)/Lib/ensurepip/_bundled/pip-$(PYTHON3_PIP_VERSION)-py3-none-any.whl
-	# FIXME: [1] get rid of this asap; 'patch' doesn't like Windows endings, and this file is full of them...
```
Comment (BKPepe) [inline lang/python/python3/Makefile:209]: We dont need this anymore?

**Comment by BKPepe** [comment]
I had just one small thing and otherwise it looks good to me. Thanks!

File: lang/python/python3/Makefile (hunk 1 above)
Comment (jefferyto) [reply lang/python/python3/Makefile:209]: This workaround isn't needed (the patched file no longer contains Windows line endings).

Comment (BKPepe) [reply lang/python/python3/Makefile:209]: Okay! Thanks. Merging.

**Review by BKPepe — APPROVED** [review]

**Pull request merged**